src/components/Admin/index.jsx

```diff
@@ -112,10 +129,24 @@ class Admin extends React.Component {
 
   getCsvErrorMessage(id) {
     const { csv } = this.props;
-    if (csv && csv[id] && csv[id].csvError) {
-      return csv[id].csvError.message;
+    const csvData = csv && csv[id];
+    return csvData && csvData.csvError;
+  }
+
+  getTableData(id = 'enrollments') {
+    const { table } = this.props;
+    const tableData = table && table[id];
+    return tableData && tableData.data;
+  }
+
+  shouldDisableCsvButton(id) {
+    const tableData = this.getTableData(id);
+    if (!tableData) {
+      return true;
     }
-    return '';
+    const isTableLoading = tableData.loading;
+    const isTableEmpty = tableData.results && !tableData.results.length;
```

Comment:
Was the empty table check necessary? Looks like it is already captured on the row with `this.hasEmptyData()` https://github.com/edx/edx-portal/pull/82/files#diff-a3ca57fe250c904aa8b89703ffc82c38R262

Doesn't hurt to have it here too, just curious.

Reply:
`this.hasEmptyData()` is checking if the overview data is empty, not the table. Some of the table data could be empty even if the overview data is not empty. Therefore, it's necessary to check if the table data is empty on its own to determine whether the CSV button should be disabled.